javadoc/javadoc-all/pom.xml

```diff
@@ -235,6 +235,11 @@
          <groupId>org.infinispan</groupId>
          <artifactId>infinispan-scripting</artifactId>
       </dependency>
+
+      <dependency>
```

Comment:
have you tried to exclude the package?
See https://github.com/infinispan/infinispan/blob/master/javadoc/javadoc-all/pom.xml#L311

it seems a better fix than add it as a dependency. AFAIK, it is only a test dependency, right?

Reply:
@pruivo ideally our users would also use BlockHound and our integration would warn them when they use blocking methods in callbacks that should be non-blocking, like listeners, or even `cache.putAsync(key, value).thenAccept(__ -> cache.put(key2, value2)`

Reply:
Actually the more I think about it we may want to include the BlockHoundIntegration in the Javadoc so users can be aware of it. Thus I would say to keep this PR as is.